tests/integration/devfile/cmd_devfile_url_test.go

```diff
@@ -140,19 +129,14 @@ var _ = Describe("odo devfile url command tests", func() {
 			helper.CmdShouldPass("odo", "create", "nodejs", "--project", namespace, componentName)
 
 			helper.CopyExample(filepath.Join("source", "devfiles", "nodejs", "project"), context)
-			helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfile.yaml"), filepath.Join(context, "devfile.yaml"))
+			helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfile-with-multiple-endpoints.yaml"), filepath.Join(context, "devfile.yaml"))
 
-			helper.CmdShouldPass("odo", "url", "create", url1, "--port", "3000", "--host", host, "--secure", "--ingress")
+			helper.CmdShouldPass("odo", "url", "create", url1, "--port", "9090", "--host", host, "--ingress")
```

Comment:
Do we really need this spec? I can see secure url is created and verified in https://github.com/openshift/odo/pull/3665/files#diff-5b34c0d9e5819031b99fc98f27ed1690R100 spec. I would say lets add `odo url list` also in that spec. No need to create a separate spec for this

Reply:
same for https://github.com/openshift/odo/pull/3665#discussion_r466633933